eos_data_distribution/ndn/command.py

```diff
@@ -70,22 +70,39 @@ def makeInterest(cmd, local=True, controlParameters={},
 def main():
     from gi.repository import GLib
     from . import base
-    import argparse
+    from tests import utils
 
     logging.basicConfig(level=logging.DEBUG)
 
-    parser = argparse.ArgumentParser(description='Command Interest Tests')
-    parser.add_argument("-n", "--name")
+    parser = utils.process_args(description='Command Interest Tests')
+    parser.add_argument("--faceid", "-i")
+    parser.add_argument("--uri", "-u")
+    parser.add_argument("--local-control-feature", "-l")
+    parser.add_argument("--origin", "-o")
+    parser.add_argument("--cost", "-c")
```

Comment:
`type=int` here will do the type checking for you. Similarly for the other arguments.